packages/amplify-util-uibuilder/src/index.ts

```diff
@@ -1,16 +1,26 @@
 import { $TSContext } from 'amplify-cli-core';
 import path from 'path';
 
-export async function executeAmplifyCommand(context: $TSContext, cmd: string) {
+// TODO: remove dynamic require
```

Comment:
is this intended to be done in this PR?

Reply:
nope we want to scope that in different pr